3696/1/gem_info.rb

```diff
@@ -0,0 +1,37 @@
+require './scraper'
+# The same as top_gems.rb(especially in popularity)
+# :reek:InstanceVariableAssumption and :reek:TooManyInstanceVariables
+class GemInfo
+  include Scraper
+  attr_reader :used_by, :watched_by, :starred_by, :forked_by, :contributors, :issues, :name
+  def to_s
+    "#{@name}  | used by #{@used_by} | watched by #{@watched_by} |  #{@starred_by} stars |"\
+    "#{@forked_by} forks | #{@contributors} contributors | #{@issues} issues |"
+  end
+
+  def get_original_page_info(link)
```

Comment:
we don't usually add get_ set_ prefixes in ruby

Reply:
scrape_ would be better?

Reply:
just `original_page_info`